src/plugins/intel_gpu/src/graph/program_dump_graph.cpp

```diff
@@ -136,7 +136,16 @@ void close_stream(std::ofstream& graph) { graph.close(); }
 
 std::string get_node_id(const program_node* ptr) { return "node_" + std::to_string(reinterpret_cast<uintptr_t>(ptr)); }
 
-void dump_full_node(std::ofstream& out, const program_node* node) { out << node->type()->to_string(*node); }
+void dump_full_node(std::ofstream& out, const program_node* node) {
+    GPU_DEBUG_GET_INSTANCE(debug_config);
+    try {
+        out << node->type()->to_string(*node);
+    } catch(const std::exception& e) {
+        GPU_DEBUG_IF(debug_config->verbose >= 2) {
+            std::cerr << node->id() << " to_string() error: " << e.what() << '\n';
+        }
+    }
```

Comment:
Instead of just skiping could we add the node to the graph with empty properties?

Reply:
Good idea, applied it.

Reply:
1. What about:
```
GPU_DEBUG_INFO << node->id() << " to_string() error: " << e.what() << '\n';
```
You don't need `GPU_DEBUG_IF`

2. Instead of "empty" what about adding an error message there too so that user can understand it failed? Such as "failed to make string from descriptor"